### ISSUE TITLE
MeatPI reports ~30V instead of ~12V

### ISSUE THREAD

**Issue by eludias** [meatpiHQ/wican-fw#54]
The web-interface reports 28V - 31V as the 'Battery Voltage' on WiCAN-OBD in my Chevy Bolt EV instead of the 'normal' 12V.

Reading out the voltages with an external voltage meter shows around ~12V at all pins of the CAN bus.

Is this a software bug and/or EV related?

**Comment by RichardPar** [comment]
![image](https://github.com/meatpiHQ/wican-fw/assets/1158641/50e055a9-ea3b-4c70-ab87-47fbfc2140ae)

Same ..

**Comment by meatpiHQ** [comment]
@RichardPar can you please post a screenshot of the about page?

**Comment by RichardPar** [comment]
> @RichardPar can you please post a screenshot of the about page?

![image](https://github.com/meatpiHQ/wican-fw/assets/1158641/fe6d79a2-5f61-46b0-833b-28868861ee2a)

**Comment by meatpiHQ** [comment]
@RichardPar @eludias It seems you accidently flashed the "WiCAN-USB" firmware? Please try to flash the OBD version.

WiCAN-USB can operate at higher voltage, so it has a different divider value.

![image](https://github.com/meatpiHQ/wican-fw/assets/94690098/01e87e1a-f251-44f8-b78d-105d125c8516)

**Comment by RichardPar** [comment]
> @RichardPar @eludias It seems you accidently flashed the "WiCAN-USB" firmware? Please try to flash the OBD version.
> 
> WiCAN-USB can operate at higher voltage, so it has a different divider value.
> 
> ![image](https://user-images.githubusercontent.com/94690098/247103973-01e87e1a-f251-44f8-b78d-105d125c8516.png)

Thanks...

Can the Documentation be made clearer on the different ones? or even an image with the link for firmware next to it?

**Comment by eludias** [comment]
Yes, I've flashed the wrong image. New OBD image reports correct voltage.

Thanks for the support!
